packages/radix-ui-themes/changelog.md

```diff
@@ -24,6 +24,8 @@
     - `flexBasis`, `flexShrink`, `flexGrow`
     - `gridColumn`, `gridColumnStart`, `gridColumnEnd`
     - `gridRow`, `gridRowStart`, `gridRowEnd`
+  - **[Breaking]** The `width` and `height` props don't map to space scale anymore. This is because in the vast majority of cases, width and height were not set to space scale, and with that, space scale as an IDE autocomplete suggestion felt odd/misleading.
+    - **[Upgrade guide]** Find and replace your `width` and `height` prop usage with the corresponding [space scale](https://github.com/radix-ui/themes/blob/main/packages/radix-ui-themes/src/styles/tokens/space.css) steps, e.g. `width="9"` would need to be changed to `width="64px"`, and so on for other steps.
```

Comment:
Should we suggest they could also refer to the CSS variables?

Reply:
Good point, will add!